eng/targets/Settings.props

```diff
@@ -43,6 +43,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(NoGenerateSatelliteAssemblies)' == 'true'">
```

Comment:
>NoGenerateSatelliteAssemblies [](http://example.com/codeflow?start=31&length=29)

You can set `/p:EnableXlfLocalization=false` on command line directly rather then adding `NoGenerateSatelliteAssemblies`

Reply:
This is not correct the `CoreGenerateSatelliteAssemblies` targets still run even if `EnableXlfLocalization` is set to false. Since a significant number of the time savings comes from not running these targets we still need some explicit property.

Reply:
I could refactor this to tie everything into the `EnableXlfLocalization` property instead of creating a new one.

Reply:
I'd avoid redefining/repurposing of `EnableXlfLocalization` - this property is defined by Microsoft.DotNet.XliffTasks and we should not change its semantics.

Reply:
alright, I've tied this behavior to a new property: `GenerateSatelliteAssemblies`

Reply:
Will our unit test work w/o satellite assemblies? We do test localized error messages...